### ISSUE TITLE
Accessibility issues with logo SVG

### ISSUE THREAD

**Issue by alisonrae** [PublicisSapient/accessible-ecommerce-demo#33]
1. Remove the title attribute from the link containing the svg. The text read to the screen reader user should be the same as the text created in the svg.
2. svg needs the attribute focusable="false" to work properly for IE and JAWS. Please add this attribute.

**Comment by mikequattrin** [comment]
Fix has been applied for this issue. Closing.
